Assets/Scripts/Cat Scripts/MovementController.cs

```diff
@@ -109,9 +127,39 @@ private void RotateCharacter()
             Quaternion targetRotation = Quaternion.LookRotation(moveDirection, Vector3.up);
 
             // smooth interpolate between the actual rotation to target rotation 
-            //we use .Slerp because the movement is smoother than other functions that are linear
+            //we use .Slerp because the movement is smoother than other linear functions 
```

Comment:
Comments on line 130 are written in the first person, which is not suitable for professional documentation. Comments should be neutral and consistent.

Reply:
the commentary is written in the third person, for greater familiarity with the code.